src/pkg/selectors/onedrive.go

```diff
@@ -459,6 +459,29 @@ func (s OneDriveScope) setDefaults() {
 	}
 }
 
+// DiscreteCopy makes a clone of the scope, then replaces the clone's user comparison
+// with only the provided user.
+func (s OneDriveScope) DiscreteCopy(user string) OneDriveScope {
+	return discreteCopy(s, user)
+}
+
+// ---------------------------------------------------------------------------
+// Backup Details Filtering
+// ---------------------------------------------------------------------------
+
+// Reduce filters the entries in a details struct to only those that match the
+// inclusions, filters, and exclusions in the selector.
+func (s oneDrive) Reduce(ctx context.Context, deets *details.Details) *details.Details {
```

Comment:
Weird diff: this func wasn't changed.  What changed is that `matchesInfo()` was moved below it.